src/main/java/org/scalameter/annotations/benchmark.java

```diff
@@ -0,0 +1,31 @@
+package org.scalameter.annotations;
```

Comment:
No, it could happen when user would use wildcard import on japi to import all the annotations . So this way they would have also old Java API classes imported (like Warmer). 
On the other hand, in the new annotation based frontend, they just use normal ScalaMeter classes, so to specify warmer, they just override `def warmer: org.scalameter.Warmer` and not `org.scalameter.japi.Warmer`. Hence we have a conflict here.